plotly/tools.py

```diff
@@ -3881,6 +3881,9 @@ def _scatterplot(dataframe, headers, diag, size,
             title=title,
             showlegend=True
         )
+
+        fig = FigureFactory._hide_tick_labels_from_box_subplots(fig)
```

Comment:
Can you rename this to something more generic now? I.e., the `box` in there? Just looks funny to me.